src/Dialog/Dialog.tsx

```diff
@@ -421,9 +504,11 @@ const DialogCloseButton = styled(Button)`
   color: ${get('colors.fg.muted')};
   padding: ${get('space.2')};
   align-self: flex-start;
+  pointer-events: auto; // Allows clicking when combined with a draggable bottom-sheet header
```

Comment:
What happens if we don't set a `pointer-events` style at all? I'd assume it would be the same as `pointer-events: auto;` unless we style `pointer-events` somewhere else that I missed.

Reply:
Sadly if I remove it doesn't work anymore.